ktlint-rule-engine/src/main/kotlin/com/pinterest/ktlint/rule/engine/internal/SuppressionLocatorBuilder.kt

```diff
@@ -207,6 +208,9 @@ internal object SuppressionLocatorBuilder {
                 // not prefixed with any rule set id.
                 RuleId.prefixWithStandardRuleSetIdWhenMissing(it)
             }
+            .filter {
+                IdNamingPolicy.matchesRuleIdRegex(it)
```

Comment:
Hi, @paul-dingemans do you know how to log a warning here? Then I will create a `fun matchesRuleIdRegexAndLog` which will return the boolean and also log the warning. Is there any dependency of logger here?

Reply:
Did you already try to add `private val LOGGER = KotlinLogging.logger {}.initKtLintKLogger()` as top level declaration?

I don't think that you should create a method `matchesRuleIdRegexAndLog` as that would duplicate the regex logic that only should live in the `IdNamingPolicy`. Maybe just a `try-catch`?

Reply:
Updated my logic